packages/ffe-account-selector-react/src/subcomponents/input-field/InputField.js

```diff
@@ -1,147 +1,114 @@
 /* Because we are missing aria-controls (http://www.heydonworks.com/article/aria-controls-is-poop): */
 /* eslint jsx-a11y/role-has-required-aria-props:0 */
-import React, { Component } from 'react';
+import React from 'react';
 import { func, string, bool, number } from 'prop-types';
-import autoBind from 'react-auto-bind';
-import KryssIkon from '@sb1/ffe-icons-react/lib/kryss-ikon';
-import ChevronIkon from '@sb1/ffe-icons-react/lib/chevron-ikon';
+import { KryssIkon, ChevronIkon } from '@sb1/ffe-icons-react';
 import classNames from 'classnames';
 import txt from '../../i18n/i18n';
 import { Locale } from '../../util/types';
 
-class Input extends Component {
-    constructor(props) {
-        super(props);
-        autoBind(this);
-        this.state = {
-            value: props.value,
-            isFocused: false,
-        };
-    }
+const InputField = props => {
+    const {
+        onKeyDown,
+        id,
+        placeholder,
+        isSuggestionsShowing,
+        ariaInvalid,
+        onClick,
+        inputFieldRef,
+        highlightedIndex,
+        suggestionListId,
+        name,
+        readOnly,
+        locale,
+        value,
+        onBlur,
+        onFocus,
+    } = props;
+    const onChange = ({ target: { value: newValue } }) =>
+        props.onChange(newValue);
 
-    onChange(e) {
-        const value = e.target.value;
-        this.setState({ value });
-        this.props.onChange(value);
-    }
-
-    onFocus() {
-        this.setState({ isFocused: true });
-        this.props.onFocus();
-    }
-
-    onBlur() {
-        this.setState({ isFocused: false });
-        this.props.onBlur();
-    }
-
-    onReset(e) {
+    const onReset = e => {
```

Comment:
onReset -> handleReset?
Blir vell samma med de on... andra hvis du har lust.

Reply:
Yup, that should be fine. This way I can deconstruct everything from props.

It does mean you have to approve the PR again though ;)